reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -45,6 +45,7 @@
       organizations: "{{ github_organization }}"
     osm_use_cockpit: true
```

Comment:
always have by default. Some of the consultants have plugged it in and demo'd it in the field